main.js

```diff
@@ -160,4 +160,177 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+
+function countInventoryToSell (televisions) {
+  console.log("SALES", televisions);
+  let totalCount = 0;
+
+  for (const television of televisions) {
+    console.log("television in loop", television.originalStock, television.sold);
+
+    const toSellOfThisType = television.originalStock - television.sold;
+    console.log("to sell of this type: ", toSellOfThisType);
+    totalCount = totalCount + toSellOfThisType;
+  }
+  console.log("Total count:", totalCount);
+  return totalCount;
+}
+
+const totalCount = countInventoryToSell(inventory);
+
+
+
+const numberToSell = document.getElementById('number-to-sell');
+numberToSell.textContent = totalCount;
+
+console.log(numberToSell);
+
+
+// Opdracht 2a: Gebruik een array-methode om een array te maken met alle tv-type namen.
+
+const tvNames = inventory.map((name) => {
+  return name.brand + " " + name.name;
+});
+console.log(tvNames);
+
+// Opdracht 2b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn.
+
+
+const soldOut = inventory.filter((television) =>{
+  return television.originalStock === television.sold;
+});
+
+// Opdracht 2c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken.
+
+const ambiLight = inventory.filter((hasAmbiLight)=> {
+  return hasAmbiLight.options.ambiLight === true;
+})
+
+console.log(ambiLight);
+
+// Opdracht 2d: Schrijf een functie die alle tv's van laagste naar hoogste prijs sorteert.
+
+const priceSortLowToHigh = inventory.sort((a, b)=> {
+  return a.price - b.price});
+
+console.log(priceSortLowToHigh);
+
+// Opdracht 3
+// Opdracht 3a: Wat is onze doel-opbrengst? Bereken wat de totale opbrengst is, als we alle exemplaren
+// van ieder type zouden verkopen. Geef dit in het blauw weer op de pagina.
+
+
+const tvPrices = inventory.map((price) => {
+  return price.price * (price.originalStock - price.sold);
+});
+console.log(tvPrices);
+
+const totalRevenue = tvPrices.reduce(function (a, b)
+{ return a + b}, 0);
```

Comment:
In principe goed gedaan, maar `function` is overbodig en de `{ }` vind ik ook niet netjes staan. Ik ben wel heel blij dat je de `.reduce` hebt gebruikt!

Als je maar één regel code nodig hebt in de functie body, dan kun je de `{...}` weglaten, zoals:
```
const totalRevenue = tvPrices.reduce((a, b) => a + b, 0);
```